MaterialComponentsCatalog.podspec

```diff
@@ -8,7 +8,7 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/material-components/material-components-ios.git", :tag => "v#{s.version}" }
   s.platform     = :ios, '8.0'
   s.requires_arc = true
-  s.source_files = 'components/*/examples/*.{h,m,swift}', 'components/*/examples/supplemental/*.{h,m,swift}'
+  s.source_files = 'components/*/examples/*.{h,m,swift}', 'components/*/examples/supplemental/*.{h,m,swift}', 'catalog/CBCBase/*.{h,m,swift}'
```

Comment:
If the source is in catalog it should likely live in MDCCatalog.xcodeproj.
This podspec is intended to pull in source from components/